docs/index.md

```diff
@@ -0,0 +1,44 @@
+# Small
+
+> C++ Small Containers
+
+[![Small](img/small_banner.svg)](https://alandefreitas.github.io/small/)
```

Comment:
Why was `small_banner.svg` deleted?

Reply:
By the way, we do need to improve this banner with something nicer with GIMP :)

Reply:
Omg, I made a blunder on that. I'll fix it.